pkg/apis/networking/types.go

```diff
@@ -55,6 +55,16 @@ type NetworkPolicySpec struct {
 	// solely to ensure that the pods it selects are isolated by default)
 	// +optional
 	Ingress []NetworkPolicyIngressRule
+
+	// List of egress rules to be applied to the selected pods. Outgoing traffic is
+	//allowed unbounded if there are no NetworkPolicies selecting the pod
+	// (and cluster policy otherwise allows the traffic), OR if the traffic source is
+	// the pod's local node, OR if the traffic matches at least one egress rule
```

Comment:
"if the traffic source is the pod's local node" is meaningless here. You might have meant to change it to "if the traffic *destination* is the pod's local node", but I don't think we need that exception here. (It exists in ingress rules to make sure that the kubelet can do health checks on pods.)

Reply:
I agree we do not need the excption, unless someone can say exactly why we need it.